dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventQuery.java

```diff
@@ -95,32 +88,4 @@ public String getColumnName() {
           "getColumnName can only be invoked on TableColumn or Function");
     }
   }
-
```

Comment:
It is not easy to understand from the PR. Where did you move this code?

Reply:
It is unused so I've removed it. We can still restore it if we need it. I guess it has been moved in the past from /event/EventQuery to trackedEntity/aggregates/query/EventQuery